src/docs/asciidoc/chapters/dedication.adoc

```diff
@@ -1,4 +1,5 @@
 = Dedication
+:page-pagination: next
```

Comment:
Why is this necessary?

Reply:
It's not and we can remove it.
The idea was to showcase how we can control the pagination to remove/hide the previous and/or next link.
In this case, only the next link will be shown.

Do you want to remove it?